hazelcast/src/main/java/com/hazelcast/logging/Tracing.java

```diff
@@ -0,0 +1,36 @@
+package com.hazelcast.logging;
+
+/**
+ * Support class for advanced troubleshooting. It's not intended to be used under normal circumstances and
+ * it is disabled by default. Set a system property {@code hazelcast.troubleshooting.tracing.enabled}
+ * to {@code true} to enable tracing.
+ *
+ * Each probe must prefix its output to by a unique code for convenient filtering and searching.
+ *
+ */
+public final class Tracing {
```

Comment:
Perhaps and idea to have a trace config. And I would like to change attributes dynamically because often in production systems you want to see what is going on and restarting is not always an option.